### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by matheuscmelo**
This PR is able to add new test structure, running isolated tests using Docker-in-Docker

### CODE REVIEW

File: assets/payload.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "plugin": "kubejobs",
+    "plugin_info": {
+        "username": "admin",
+        "password": "grafana123123",
+        "cmd": [
+            "python",
+            "application.py"
+        ],
+        "img": "matheusmelo/sort_strings_no_swift",
```
Comment (matheuscmelo) [inline assets/payload.json:10]: We could use this image built by me or other one just for the integration tests

File: assets/payload.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        "init_size": 1,
+        "redis_workload": "https://gist.githubusercontent.com/GabrielSVinha/cf2ca2604160aedbc7675aac60cccc77/raw/47b655333742d4fe0a403b6fba48137527034d62/workload",
```
Comment (matheuscmelo) [inline assets/payload.json]: Same as above, we could use a shorter workload so the cluster won't be overloaded

File: helpers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    """
+    job_payload = get_job_payload()
+    response = requests.post(manager_url + '/submissions', json=job_payload)
+    response_payload = response.json()
+    job_id = response_payload.get('job_id')
+    return job_id
+
+
+def stop_job(manager_url, job_id):
+    #TODO: discuss a better way of stopping jobs or solve the manager's unwanted behavior
```
Comment (matheuscmelo) [inline helpers/__init__.py:51]: Left this here to remember about this issue.

File: assets/payload.json (hunk 2 above)
Comment (clenimar) [reply assets/payload.json]: +1 for a shorter workload. what's the estimated time for this amount of items? (just to make sure `expected_time` and `max_replicas` also make sense here).

**Review by clenimar — APPROVED** [review]
that's a huge contribution. thanks!

File: assets/payload.json (hunk 2 above)
Comment (matheuscmelo) [reply assets/payload.json]: @clenimar I believe it's around 2mins. We should discuss this whole json file to make it adequate for this use case.

**Comment by matheuscmelo** [comment]
You can see results from these tests here http://150.165.15.68:50000/blue/organizations/jenkins/asperathos-manager/detail/integration-tests/77/pipeline

File: assets/payload.json (hunk 2 above)
Comment (matheuscmelo) [reply assets/payload.json]: Done.

File: helpers/__init__.py (hunk 3 above)
Comment (matheuscmelo) [reply helpers/__init__.py:51]: https://github.com/ufcg-lsd/asperathos-manager/issues/27

**Review by liliarsampaio — APPROVED** [review]
thanks, we really needed this! lgtm.

**Pull request merged**